src/components/StarIcon/StarIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./StarIcon.css";
+
+type StarIconProps = {
+  currentColor: string;
+  className: string;
```

Comment:
необязательный пропс, который даже не используется
убираем

```suggestion
  className?: string;
```